pages/glossary/hidden.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Explicitly Hidden
```

Comment:
Thinking about this a little more... I don't think I like having a "hidden" definition that isn't the opposite of the "visible" definition. We did that in axe-core and it's pretty confusing. I think we should rename this to "hidden state", and say it can be either true or false.

Reply:
Which is why I called it "explicitly hidden" in the first place https://github.com/act-rules/act-rules.github.io/pull/1189#discussion_r397726136 😆 
"hidden state" is also a good name.